dimagi/utils/couch/pagination.py

```diff
@@ -258,7 +258,7 @@ def get_ajax_response(self, request, search_query, extras={}):
 
         to_return = {"sEcho": params.echo,
                      "iTotalDisplayRecords": total_rows,
-                     "aaData": all_json}
+                     "aaData": list(all_json)}
         
         for key, val in extras.items():
             to_return[key] = val
```

Comment:
it's still a problem. you never know who's going to modify this function later and always good to be defensive. thanks for the fix!